Riot/Modules/People/Views/InviteRecentTableViewCell.m

```diff
@@ -42,14 +43,12 @@ - (void)awakeFromNib
     [self.leftButton.layer setCornerRadius:5];
     self.leftButton.clipsToBounds = YES;
     [self.leftButton setTitle:NSLocalizedStringFromTable(@"decline", @"Vector", nil) forState:UIControlStateNormal];
-    [self.leftButton setTitle:NSLocalizedStringFromTable(@"decline", @"Vector", nil) forState:UIControlStateHighlighted];
     [self.leftButton addTarget:self action:@selector(onDeclinePressed:) forControlEvents:UIControlEventTouchUpInside];
     
     [self.rightButton.layer setCornerRadius:5];
     self.rightButton.clipsToBounds = YES;
-    [self.rightButton setTitle:NSLocalizedStringFromTable(@"preview", @"Vector", nil) forState:UIControlStateNormal];
-    [self.rightButton setTitle:NSLocalizedStringFromTable(@"preview", @"Vector", nil) forState:UIControlStateHighlighted];
-    [self.rightButton addTarget:self action:@selector(onPreviewPressed:) forControlEvents:UIControlEventTouchUpInside];
+    [self.rightButton setTitle:NSLocalizedStringFromTable(@"accept", @"Vector", nil) forState:UIControlStateNormal];
```

Comment:
We still want to preview rooms. We want to directly accept only DM.

Reply:
It's handled like Android you need to tap background cell. Preview action in `[RecentsViewController tableView:didSelectRowAtIndexPath:]`